src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyTestDefault3.java

```diff
@@ -78,28 +78,29 @@ else if (x < 0) { // ok
 
     // Test static initialiser
     static
-    { // violation
+    { // violation ''{' at column 5 should be on the previous line'
         int x = 1; // should not require any javadoc
     }
 
 
 
     public enum GreetingsEnum
-    { // violation
+    { // violation ''{' at column 5 should be on the previous line'
         HELLO,
         GOODBYE
     };
 
     void method2()
-    { // violation
+    { // violation ''{' at column 5 should be on the previous line'
         boolean flag = true;
         if (flag) { // ok
             System.identityHashCode("heh");
             flag = !flag; } String.CASE_INSENSITIVE_ORDER.
               equals("Xe-xe");
         // it is ok to have rcurly on the same line as previous
         // statement if lcurly on the same line.
-        if (flag) { String.CASE_INSENSITIVE_ORDER.equals("it is ok."); } // violation
+        if (flag) { String.CASE_INSENSITIVE_ORDER.equals("it is ok."); }
+        // violation above ''{' at column 19 should have line break after'
```

Comment:
Remove 2 comments above and do `violation below `

Reply:
Done